### ISSUE TITLE
Kandy.call.createVoipCall crashes the the app

### ISSUE THREAD

**Issue by primableatom** [Kandy-IO/kandy-phonegap#35]
This is what I did 

    Kandy.call.createVoipCall(
        function(){
           ..////
        },
         function(e){
             error(e);
          },
          callee,
          true
        );

This crashed the caller app but the callee received the call. 
Happened in iOS 8.3

**Comment by srbaskar** [comment]
Hi,

I'm not able to re-produce this error while making a VOIP call between device 1 (iOS 8.3) to device 2 (iOS 9.1).

We'll have a screen-sharing session and fix the issue.

Thanks,
Srini

**Comment by primableatom** [comment]
Hey Srini,

This is a screen cast of the crash issue https://www.dropbox.com/s/kzyd4tca2weuhe9/kandy_call_issue.mp4?dl=0

As I said its crashing when I try to enableVideo and unmute programatically on success of createVoipCall

**Comment by srbaskar** [comment]
Hi Anup,

I've fixed the crash issue and pushed the fix into Github.

Please pull the latest changes and test it and let me know if you have any question.

Thanks,
Srini

**Comment by primableatom** [comment]
Hey Srini,

Its not crashing any more. I tried it a few times and noticed the when user1 makes a video call to user2
and after user2 accepts the call with "Answer with Video" it works fine for user2 BUT user1 is sometimes not able to see user2 video. I tried it around 10 times and it was only once when I (user1) got to see user2. May be a network issue at my end. I will test it a little more evening and let you know

Thanks,
Anup
